src/functions/versions.ts

```diff
@@ -29,7 +29,7 @@ const migrate = (functionsPath: string, verbose = false): void => {
       postfix = `-${version}`;
     } else {
       version = '1.0';
-      postfix = ` => ${name}/${version}`;
```

Comment:
Can you use path.join here instead of path.sep? 
Makes it more consistent and means you don't have to think about the separator.